### PR TITLE
Changes to notifications and job preferences

### LINKED ISSUE #210: Enhancement : Display job preferences of user and notify them when saving preferences
list the preferences of the user at the top of the page and alert them when they save preference

### PR DESCRIPTION
**Pull request by DasDeveloper**
- The user can now see his preferences that were already set before on the job listing page.
image

- The user will see "no jobs found based on your preferences" as an alert if there are no jobs found based on the user's preferences. 
image

### CODE REVIEW

File: src/client/src/components/JobListing.jsx [hunk 1]
```diff
@@ -107,19 +109,33 @@ const JobListing = () => {
   const savePreferences = async () => {
     if (preferences.category && preferences.location && preferences.work_type) {
       await axios.post(`http://localhost:9000/users/${userID}/preferences`, preferences);
+          setTimeout(() => {
+          swal("Preferences saved successfully!.");
+      window.location.reload();
```
Comment (imran1289-ah) [inline src/client/src/components/JobListing.jsx:114]: Can you add the translation for this alert.

**Review by imran1289-ah — APPROVED** [review]
make sure to add the translation in the i18next file, but other than the code seems fine.

**Review by imran1289-ah — CHANGES_REQUESTED** [review]
all is good, just need to add the translation in i18next file, if not done

**Comment by imran1289-ah** [comment]
make sure to test out that the translation displays once the switch to French

File: src/client/src/components/UserTimeline.jsx [hunk 2]
```diff
@@ -220,7 +220,14 @@ const UserTimeline = () => {
...
     });
   };
-  const notify = () => toast(t('New Job Alert Posted: ') + job.title);
+  const notify = () => {
+    if(job!=null){
+    toast(t('New Job Alert Posted: ') + job.title);
+    }
+    else{
+      toast(t('No jobs found based on your preferences'))
+    }
```
Comment (imran1289-ah) [inline src/client/src/components/UserTimeline.jsx:229]: just need to add this translation in i18next file

Comment (imran1289-ah) [reply src/client/src/components/UserTimeline.jsx:229]: from the code the rest seems good, i didnt test out the feature but i can see from the code it should work. i think u can merge it once you checked the translation

Comment (Kunal22shah) [reply src/client/src/components/UserTimeline.jsx:229]: added, i'll merge the PR now. Thanks!

**Pull request merged**